### DISCUSSION TITLE
type unknown error in getDisplayName

### DISCUSSION THREAD

**Discussion by Aledosim** [cypress-io/cypress#15688] [Component Testing]
I executed ```npx create-cypress-tests --component-tests``` and when tried to run a component test it threw this error. Using create-react-app and plain js. My E2E tests are ok. Is this a configuration error? What's the displayName?
Here's the test:
```
import React from 'react'
import { mount  } from '@cypress/react'
import { Background  } from './Background'

it('renders without crashing', () => {
    mount(<Background />)

})

```

**Comment by lmiller1990** [comment]
Hey, sorry for the slow rely.

Can you share the generated code in `cypress/plugins/index.js`? `create-cypress-tests` is still a work in progress, it might have generated the incorrect config.
